clientsampleapp/build.gradle

```diff
@@ -46,7 +46,8 @@ dependencies {
     implementation 'com.google.android.material:material:1.10.0'
     implementation 'androidx.appcompat:appcompat:1.6.1'
     implementation 'com.github.BirjuVachhani:locus-android:3.0.1'
-    implementation 'com.github.blinkupsdk:bLinkupAndroidSDK:2.0.7'
+//    implementation 'com.github.blinkupsdk:bLinkupAndroidSDK:2.0.7'
+    implementation project(':sdk')
```

Comment:
Same as the other PR do you mean to remove the package and only use the project reference?